aptos-move/block-executor/src/executor.rs

```diff
@@ -412,7 +385,10 @@ where
             // Must contain committed value as we set the base value above.
             delta_writes.push((k, WriteOp::Modification(serialize(&committed_delta).into())));
         }
+
         last_input_output.record_delta_writes(txn_idx, delta_writes);
+
+        // TODO(zi) Make sure this here is perfectly reentrant, and doesn't introduce regression.
```

Comment:
not sure what is comment is exactly worried about, and whether you plan to address it before landing this PR?

Reply:
That was a not to myself - was not suppose to stay here. Thanks for catching it.